src/metaopt/io/database/abstractdb.py

```diff
@@ -32,19 +32,19 @@ def close_connection(self):
         pass
 
     @abstractmethod
-    def insert(self, experiment_name, trial_field, dictionary, field_filters):
+    def insert(self, collection_name, dictionary, field_filters):
         pass
 
     @abstractmethod
-    def update(self, experiment_name, trial_field, update_dictionary, field_filters):
+    def update(self, collection_name, update_dictionary, field_filters):
```

Comment:
I think `(self, collection_name, query, update_dictionary)` would be more clear.